platform/lib/schema/index.ts

```diff
@@ -436,6 +436,65 @@ export class ObjectSetting<P extends Props> extends Setting<
   }
 }
 
+function isMap<K, V>(o: any): o is Map<K, V> {
```

Comment:
🙈 "search internet, copy first answer" didn't work :partyparrot: <--- github needs to add this asap